tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -9625,6 +9658,449 @@ Curve:QuadLinear,
+
+    state->dataGlobal->SysSizingCalc = false; // DISABLE SIZING - don't call UnitarySystem::sizeSystem, much more work needed to set up sizing arrays
+    state->dataHVACGlobal->DoSetPointTest = true; // Force setpoint check
+
+    auto &inletNode = state->dataLoopNodes->Node(thisSys->AirInNode);
+    auto &outletNode = state->dataLoopNodes->Node(thisSys->AirOutNode);
+
+    // set up unitary system inlet condtions
+    inletNode.Temp = 26.666667;             // AHRI condition 80F dry-bulb temp
+    inletNode.HumRat = 0.01117049542334198; // AHRI condition at 80F DB/67F WB lb/lb or kg/kg
+    inletNode.Enthalpy = Psychrometrics::PsyHFnTdbW(inletNode.Temp, inletNode.HumRat);
+
+    state->dataScheduleMgr->Schedule(1).CurrentValue = 1.0;
+    state->dataGlobal->BeginEnvrnFlag = true;
+    state->dataEnvrn->StdRhoAir = Psychrometrics::PsyRhoAirFnPbTdbW(*state, 101325.0, 20.0, 0.0); // initialize RhoAir
+    inletNode.MassFlowRate = thisSys->m_DesignFanVolFlowRate * state->dataEnvrn->StdRhoAir;
+    inletNode.MassFlowRateMaxAvail = thisSys->m_DesignFanVolFlowRate * state->dataEnvrn->StdRhoAir;
+
+    OutputReportPredefined::SetPredefinedTables(*state);
+
+    int AirLoopNum = 0;
+    int CompIndex = 1;
+    bool HeatActive = false;
+    bool CoolActive = true;
+    int constexpr ZoneOAUnitNum = 0;
+    Real64 constexpr OAUCoilOutTemp = 0.0;
+    bool const ZoneEquipment = true;
+    Real64 sensOut = 0.0;
+    Real64 latOut = 0.0;
+
+    outletNode.TempSetPoint = 20.0; // expect cooling coil operation
+    thisSys->simulate(*state,
+                      thisSys->Name,
+                      FirstHVACIteration,
+                      AirLoopNum,
+                      CompIndex,
+                      HeatActive,
+                      CoolActive,
+                      ZoneOAUnitNum,
+                      OAUCoilOutTemp,
+                      ZoneEquipment,
+                      sensOut,
+                      latOut);
+
+    auto &coolingCoil = state->dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(thisSys->m_CoolingCoilIndex);
+    auto &heatingCoil = state->dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(thisSys->m_HeatingCoilIndex);
+    EXPECT_NEAR(outletNode.Temp, 20.0, 0.01);
+    EXPECT_NEAR(coolingCoil.QLoadTotal, 19885.0, 1.0);
+    EXPECT_NEAR(heatingCoil.QLoadTotal, 0.0, 1.0);
+
+    outletNode.TempSetPoint = 30.0; // expect heating coil operation
+    thisSys->simulate(*state,
+                      thisSys->Name,
+                      FirstHVACIteration,
+                      AirLoopNum,
+                      CompIndex,
+                      HeatActive,
+                      CoolActive,
+                      ZoneOAUnitNum,
+                      OAUCoilOutTemp,
+                      ZoneEquipment,
+                      sensOut,
+                      latOut);
+
+    EXPECT_NEAR(outletNode.Temp, 30.0, 0.01); // higher than inlet due to fan heat
```

Comment:
"due to fan heat"? It's a blow thru fan, SP should be hit within tolerance.

Reply:
That comment must be left over from an earlier iteration of this test. The outlet is matching the setpoint of 30.0 set a few lines before this. Comment deleted.